client/src/lib.rs

```diff
@@ -40,27 +40,35 @@ use futures::{
 };
 use futures01::sync::mpsc as mpsc01;
 use jsonrpsee_types::{
-    client::{
-        FrontToBack,
-        NotificationMessage,
-        RequestMessage,
-        Subscription,
-        SubscriptionMessage,
-    },
-    error::Error as JsonRpseeError,
-    jsonrpc::{
-        self,
-        Call,
-        DeserializeOwned,
-        Id,
-        MethodCall,
-        Notification,
-        Output,
-        Request,
-        SubscriptionId,
-        SubscriptionNotif,
-        Version,
+    v2::{
+        error::{
+            ErrorCode,
```

Comment:
This fails to compile for me locally, `ErrorCode` is not found

Reply:
yeah, I will fix it changed in the new version.